api/HTMLMediaElement.json

```diff
@@ -1003,7 +1003,7 @@
               "version_added": "5.0"
             },
             "webview_android": {
-              "version_added": "49"
+              "version_added": false
```

Comment:
Yep, confirmed by crbug.com/521319. I'll send a PR to link that issue after merging this.